src/commands/cloud-login.js

```diff
@@ -4,17 +4,21 @@ const FaunaCommand = require('../lib/fauna-command.js')
 const os = require('os')
 const fetch = require('request-promise')
 const url = require('url')
+require('dotenv').config()
 
-const SHELL_LOGIN_URL = 'https://app.fauna.com/shell_login'
+const SHELL_LOGIN_URL = process.env.FAUNA_SHELL_LOGIN_URL
```

Comment:
@BrunoQuaresma @rodrimaia How's this?

Reply:
@fauna-ryanharris and the default option?

Reply:
sorry I missed the next line haha.